cypress/e2e/tests/migration/task-manager/task_manager_drawer.test.ts

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    deleteApplicationTableRows,
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+} from "../../../../utils/utils";
+import { Analysis } from "../../../models/migration/applicationinventory/analysis";
+import { taskNotificationBadge } from "../../../views/common.view";
+
+const analyses: Analysis[] = [];
+const NUMBER_OF_APPS = 2;
+
+describe(["@tier2"], "Task drawer validation", () => {
+    before("Login", function () {
+        login();
+        cy.visit("/");
+        deleteApplicationTableRows();
+        cy.fixture("application").then((appData) => {
+            cy.fixture("analysis").then((analysisData) => {
+                for (let i = 0; i < NUMBER_OF_APPS; i++) {
+                    analyses.push(
+                        new Analysis(
+                            getRandomApplicationData("bookserverApp", {
+                                sourceData: appData["bookserver-app"],
+                            }),
+                            getRandomAnalysisData(analysisData["analysis_for_openSourceLibraries"])
+                        )
+                    );
+                }
+                analyses.forEach((analysis) => analysis.create());
+            });
+        });
+    });
+
+    it("Perform bulk analysis and validate information on task drawer", function () {
+        // Automates Polarion TC MTA-556
+        Analysis.analyzeAll(analyses[0]);
+        cy.get(taskNotificationBadge).click();
+
+        // Assert that Tech discovery tasks are listed
+        cy.get("h2.pf-v5-c-notification-drawer__list-item-header-title").each((item) => {
+            if (Cypress.$(item).text().includes("tech-discovery")) {
```

Comment:
Cypress.$(item) is just jQuery, and jQuery does not retry and does not wait until condition is met . It is better to use cy.wrap which will wait